lib/libutee/include/pta_it_notif.h

```diff
@@ -0,0 +1,40 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2023, Linaro Limited
+ */
+#ifndef __PTA_IT_NOTIF_H
+#define __PTA_IT_NOTIF_H
+
+#define PTA_IT_NOTIF_UUID { \
+		0x4461e5c7, 0xb523, 0x4b73, \
+		{ 0xac, 0xed, 0x75, 0xad, 0x2b, 0x9b, 0x59, 0xa1 } \
+	}
+/*
+ * PTA_IT_NOTIF_ACTIVATE_RESOURCE - Enable a interrupt notification
+ *
+ * [in]  params[0].value.a     Interrupt ID
```

Comment:
Please use a single term for the interrupt and also describe what it represents and how values are determined.

Reply:
OK